spec/index.bs

```diff
@@ -1615,7 +1615,7 @@ dictionary GPUBufferCopyView {
     required GPUBuffer buffer;
     GPUBufferSize offset = 0;
     required unsigned long rowPitch;
-    required unsigned long imageHeight;
+    required unsigned long zPitch; // in units of rowPitch bytes
```

Comment:
Basically imagePitchInBytes is zPitch * rowPitch? I think imageHeight is more clear, but maybe because I'm more used to it.

Reply:
Yes. My problem is 'imageHeight' sounds like the height of some image, and it really isn't at all.

Reply:
could be more to the point, like "rowsPerDepthSlice"?

Reply:
It's a little imprecise because not all of those rows are necessarily _in_ the depth slice. But it does use "depth" instead of "z" which matches elsewhere (copySize.depth) better.

Reply:
... to go in the opposite direction, maybe it would be better to have copySize have `z` and `layer` be separate concepts (and one of them has to be 1).

I don't have any clue whether 3d array textures would ever be a thing, but maybe we should design the api to allow them and totally separate those two "depth" concepts.

Reply:
Most APIs have features to convert 2D array layers to 3D texture views and vice-versa so I think it is safe to assume that depth and layers are the same thing.

Reply:
@kainino0x 
3d array textures might sound unreal, but there definitely are cube arrays

Reply:
How should cubemaps be specified? Does the 6 go into layer count or depth?

Reply:
Right now, it's the layer count.

Reply:
see also #520.